packages/search/src/search.tsx

```diff
@@ -398,6 +403,8 @@ export class Search extends React.Component< Props, State > {
 				aria-label={ this.props.__( 'Open Search', __i18n_text_domain__ ) }
 			>
 				{ ! this.props.hideOpenIcon && (
+					// `className` is accepted for some reason the intrisic attributes for SVG won't allow it (but it does work)
```

Comment:
This is an old `ts-ignore` that I never added a comment for so just fixing that now.